instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -147,6 +147,19 @@ def test_suppress_instrumentation(self):
 
         self.assert_span(num_spans=0)
 
+    def test_not_recording(self):
+        RequestsInstrumentor().uninstrument()
+        RequestsInstrumentor().instrument(
+            tracer_provider=self.original_tracer_provider
```

Comment:
Is the test relying on the `original_tracer_provider` to determine if the span is recording? It's not clear to me from the test as it's written.

Reply:
The `original_tracer_provider` is a api level tracer provider, which would eventually return a `DefaultSpan`. It's the simplest way to get a span that is not recording.

Reply:
Might be worth just adding a comment in here to note this, there's not an easy way to determine that from looking at the test.